scipy/stats/mstats_basic.py

```diff
@@ -446,7 +452,7 @@ def spearmanr(x, y, use_ties=True):
     if t is masked:
         prob = 0.
     else:
-        prob = betai(0.5*df,0.5,df/(df+t*t))
+        prob = betai(0.5*df, 0.5, df/(df + t * t))
```

Comment:
So here and in the rest of stats, this should be calling ``_betai`` not ``betai`` in order to avoid deprecation warnings.